VDA-XP-Reward/contracts/IVDAXPReward.sol

```diff
@@ -0,0 +1,112 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+interface IVDAXPReward {
+    /**
+     * @notice Claim information
+     * @param typeId - Proof type. Ex: 'gamer31'
+     * @param uniqueId - Unique Id.
+     * @param issueYear - Issued year of proof
+     * @param issueMonth - Issued month of proof
+     * @param xp - Value of xp
+     * @param signature - Proof signed by a trusted signer
+     */
+    struct ClaimInfo {
+        string typeId;
+        string uniqueId;
+        uint16 issueYear;
+        uint8 issueMonth; 
+        uint xp;
+        bytes signature;
+    }
+
+    /**
+     * @notice Emitted when the denominator for rate value is updated
+     * @param orgVal - Original value
+     * @param newVal - Updated value
+     */
+    event UpdateRateDenominator(uint32 orgVal, uint32 newVal);
+
+    /**
+     * @notice Emitted when the XP-VDA conversion rate updated
+     * @param orgRate - Original rate
+     * @param newRate - Updated rate
+     */
+    event UpdateConversionRate(uint32 orgRate, uint32 newRate);
+
+    /**
+     * @notice Emitted when reward claimd
+     * @param didAddress - DID address that requested this claim
+     * @param recipient - Address that receives the reward token
+     * @param rewardAmount - Total token amount rewarded
+     * @param conversionRate - XP to Token conversion rate
+     * @param denominator - Denominator for conversion rate value
+     */
+    event ClaimedXPReward(
+        address didAddress,
+        address recipient,
+        uint rewardAmount, 
+        uint conversionRate, 
+        uint denominator
+    );
+
+    /**
+     * @notice Returns the Reward token address (= Verida Token address)
+     * @return address Token address initialized in the deployment
+     */
+    function getTokenAddress() external view returns(address);
+
+    /**
+     * @notice Returns the `VeridaDIDRegistry` contract address
+     * @return address `VeridaDIDReigstry` contract address initialized in the deployment
+     */
+    function getDIDRegistryAddress() external view returns(address);
+
+    /**
+     * @notice Get the denominator for rate values
+     * @return uint Denominator value
+     */
+    function getRateDenominator() external view returns(uint32);
+
+    /**
+     * @notice Set the denominator for rate values
```

Comment:
Please add more detail on what this is used for

Reply:
Updated, Pls check again.

Reply:
Thanks.

Is there a default value? If so, specify that here as well

Reply:
Updated.